tools/download-wheels.py

```diff
@@ -31,7 +31,7 @@ def get_wheel_names(version):
 
     """
     http = urllib3.PoolManager(cert_reqs='CERT_REQUIRED')
-    tmpl = re.compile(PREFIX + version + '.*\.whl$')
+    tmpl = re.compile(rf"{PREFIX}{version}.*.whl$")
```

Comment:
I'm not sure these two re's are equivalent... the `\` was of the original re but is not included in the new one.

Reply:
since i am using raw string we don't need the \ to escape the dot here. \ doesnt have a special meaning as per re documentation:

> The solution is to use Python’s raw string notation for regular expression patterns; backslashes are not handled in any special way in a string literal prefixed with 'r'. So r"\n" is a two-character string containing '\' and 'n', while "\n" is a one-character string containing a newline. Usually patterns will be expressed in Python code using this raw string notation.

Reply:
Don't think so either. The `.` is a special character for the regular expression and needs to be escaped when used as a period.
```
In [7]: rf"{PREFIX}{version}.*.whl$"                                            
Out[7]: '^.*numpy-1.19.0rc1.*.whl$'

In [8]: f"{PREFIX}{version}.*\.whl$"                                            
Out[8]: '^.*numpy-1.19.0rc1.*\\.whl$'
```
Or in this case, the `\` is escaped to preserve it so it will appear in the regular expression. I would suggest
```
In [9]: rf"{PREFIX}{version}.*\.whl$"                                           
Out[9]: '^.*numpy-1.19.0rc1.*\\.whl$'
```
Note that one of `rf` or `fr` didn't work in Python 3.6. I don't know if that was fixed, so maybe better to stick with the original.

Reply:
I think Ross is right, although it does not matter. The `\` in the first one had no special meaning, since `\.` is not string escape sequence. In the second one there should be a `\` to make it a literal dot instead of "any-character", although in practice I guess it doesn't matter. (using raw string is good in any case).

Reply:
I think it does matter, `\` needs to be in the raw string as well, it isn't part of the python string parsing, it is part of the re parsing command.

Reply:
I guess I got confused here, since my local tests worked fine, since it considered "." as any character. Anyway, i think i will fix this up for correctness sake. Thanks for the feedback all !

Reply:
Looks like `rf` is what we used previously, so should be good for 3.6; I recall needing to fix that somewhere before.